pkg/fanal/image/daemon/image_test.go

```diff
@@ -247,3 +248,61 @@ func Test_image_RawConfigFile(t *testing.T) {
 		})
 	}
 }
+
+func Test_image_emptyLayer(t *testing.T) {
+	tests := []struct {
+		name    string
+		history dimage.HistoryResponseItem
+		want    bool
+	}{
+		{
+			name: "size != 0",
+			history: dimage.HistoryResponseItem{
+				Size: 10,
+			},
+			want: false,
+		},
+		{
+			name: "ENV",
+			history: dimage.HistoryResponseItem{
+				CreatedBy: "/bin/sh -c #(nop) ENV TESTENV=TEST",
```

Comment:
BuiltKit doesn't have `/bin/sh -c #(nop)` prefix. I also want to add such a case.

Reply:
Done